### PR TITLE
Lmfix

### PR DESCRIPTION
**Pull request by yabirgb**

### CODE REVIEW

File: updates/6/updates.sql [hunk 1]
```diff
@@ -455,4 +455,20 @@ INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xc9BC48c72154e
...
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x9695e0114e12C0d3A3636fAb5A18e6b737529023", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x9695e0114e12C0d3A3636fAb5A18e6b737529023", "C", "DFYN Token", "DFYN", 1620371946, NULL, "dfyn-network", "DFYN", "0x9695e0114e12C0d3A3636fAb5A18e6b737529023");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x2fdF40C484b1BD6F1c214ACAC737FEDc8b03E5a8", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x2fdF40C484b1BD6F1c214ACAC737FEDc8b03E5a8", "C", "MCS", "MCS", 1623309370, NULL, "yggdrash", "YEED", "0x2fdF40C484b1BD6F1c214ACAC737FEDc8b03E5a8");
+*
+UPDATE assets SET swapped_for="_ceth_0x2fdF40C484b1BD6F1c214ACAC737FEDc8b03E5a8", cryptocompare="YEED" WHERE identifier="_ceth_0xcA2796F9F61dc7b238Aab043971e49c6164DF375";/*Introduce YEED token swap*/
```
Comment (LefterisJP) [inline updates/6/updates.sql:467]: Hmm are you sure about this?

Coingecko shows https://www.coingecko.com/en/coins/yggdrash 0xca2796f9f61dc7b238aab043971e49c6164df375 as the address of the token.

Comment (yabirgb) [reply updates/6/updates.sql:467]: I got the information from here https://www.bitfinex.com/posts/669

Comment (yabirgb) [reply updates/6/updates.sql:467]: and the announcement here https://medium.com/mcs-token-official/introducing-mcs-token-c89f9d50e08b

Comment (LefterisJP) [reply updates/6/updates.sql:467]: I see. But then they should release a new coingecko entry as this will misclassify the MCS with the YGG icon.

Comment (LefterisJP) [reply updates/6/updates.sql:467]: Anyway this can be done later when possible.

**Pull request merged**